### PR TITLE
Alert improvements

### PR DESCRIPTION
**Pull request by billmn**
Replaced JS alert with SweetAlert (http://t4t5.github.io/sweetalert/)

schermata 2015-10-01 alle 23 29 16

### CODE REVIEW

**Comment by jbrooksuk** [comment]
This is nice, thanks!

**Comment by jbrooksuk** [comment]
I'll pull this down later and check it :)

File: resources/assets/js/app.js [hunk 1]
```diff
@@ -351,3 +361,16 @@ $(function() {
             .addClass("active");
     }
 });
+
+function askConfirmation(callback) {
+    swal({
+        type: "warning",
+        title: "Confirm your action",
+        text: "Are you sure you want to do this ?",
```
Comment (wngravette) [inline resources/assets/js/app.js]: you'll have to zap that space before the ?

Comment (GrahamCampbell) [inline resources/assets/js/app.js]: good catch

Comment (billmn) [inline resources/assets/js/app.js]: Done.

**Comment by jbrooksuk** [comment]
Perfect!

**Comment by GrahamCampbell** [comment]
Feels like the yes and cancel buttons are the wrong way around.

**Comment by jbrooksuk** [comment]
They're right according to `confirm()`?

**Comment by GrahamCampbell** [comment]
Just looks wrong to me.

**Comment by GrahamCampbell** [comment]
Maybe it's just me.

**Comment by jbrooksuk** [comment]
I think it may be the colours.

**Comment by GrahamCampbell** [comment]
Maybe.

**Comment by billmn** [comment]
Colors and buttons text can be customized if you want. 

We maintain the standard JS confirm prompt?

**Pull request merged**